ui/static/ui/js/facets.js

```diff
@@ -0,0 +1,42 @@
+define('facets', ['jquery', 'bootstrap', 'icheck'], function($) {
+  'use strict';
+  var constructFacetQuery = function(facetCheckbox) {
+    // Takes a dom element and returns the string
```

Comment:
At some point we may want to use [JSDoc](http://usejsdoc.org/about-getting-started.html) but this is fine for now